### PR TITLE
added useToggle()

### PR DESCRIPTION
**Pull request by DanielOber**
**Description**

- added vueUse to npm deps
- added useToggle

**Reference**

Issues #69

### CODE REVIEW

File: refarch-frontend/src/App.vue
Comment (devtobi) [inline refarch-frontend/src/App.vue:1]: I think this file still needs some reformatting with prettier. ;)

**Review by devtobi — CHANGES_REQUESTED** [review]

File: refarch-frontend/src/App.vue [hunk 1]
```diff
@@ -101,13 +101,14 @@ import { APPSWITCHER_URL, ROUTES_GETSTARTED } from "@/Constants";
...
 import User, { UserLocalDevelopment } from "@/types/User";
+import {useToggle} from "@vueuse/core";
 
-const drawer = ref(true);
 const query = ref<string>("");
 const appswitcherBaseUrl = APPSWITCHER_URL;
 
 const snackbarStore = useSnackbarStore();
 const userStore = useUserStore();
+const [drawer, toggleDrawer] = useToggle();
```
[the remark this answers is not in the document]
Comment (DanielOber) [reply refarch-frontend/src/App.vue:111]: drawer is used inside the v-model on line 57, so it is actually needed.

[the remark this answers is not in the document]
Comment (devtobi) [reply refarch-frontend/src/App.vue:111]: Sorry I overlooked that. ;)

**Review by devtobi — APPROVED** [review]
LGTM

**Pull request merged**